pkg/deploy/setup.go

```diff
@@ -89,3 +92,14 @@ func GetPlatform(cli client.Client) (Platform, error) {
 	// check and return whether ODH or self-managed platform
 	return isSelfManaged(cli)
 }
+
+// ShouldConfigureServiceMesh determines if the operator should invoke service-mesh specific setup.
+func ShouldConfigureServiceMesh(cli client.Client, dscispec *dsci.DSCInitializationSpec) (bool, error) {
```

Comment:
I'm still a little bit dubious about this func.
Since we will be downstreaming this feature, I'm not sure what would be the platforms this should work for.

Reply:
Removed